lib/iex/lib/iex/introspection.ex

```diff
@@ -293,6 +293,7 @@ defmodule IEx.Introspection do
               module.module_info(:exports)
           end
           |> Enum.sort()
+          |> Enum.dedup()
```

Comment:
We have several occurrences of `system_info: 1` due to [multiple specs](https://www.erlang.org/doc/man/erlang.html#system_info-1)

Reply:
This may be an issue in Erlang as well, adding too many entries to the docs. :)

Reply:
Reported upstream https://github.com/erlang/otp/issues/7501